lib/storage/client.js

```diff
@@ -1196,6 +1196,42 @@ class Client extends CryptoConsumer {
     }
     return returnValues
   }
+  /**
+   *
+   * @param {string} clientID   UUID assigned by TozStore, used to identify the client; optional.
+   * @param {String} groupName Group name you want to get information regarding.
+   *
+   * @return {Object}  The group object or null if it does not exist.
+   */
+  async groupInfo(clientID = null, groupName) {
+    const urlData = {
```

Comment:
probably a bit nicer to populate the object entirely before running the `urlEncodeData()` function -- it feels a bit odd to run it and then append strings. Also, it runs your values through the encoder to make sure they are url-safe strings for final output.